src/gtl/components/data-table/data-table.html

```diff
@@ -14,9 +14,6 @@
          ng-if="tableCtrl.rows && tableCtrl.rows.length !== 0">
     <thead>
     <tr>
-      <th class="narrow">
-
-      </th>
       <th ng-if="$index !== 0"
```

Comment:
should this `ng-if` also go away? Otherwise we're still skipping the first column name, right?

Reply:
@himdel fix revised, please re-review

Reply:
@himdel, not sure how to fix failures on this PR

Reply:
@h-kataria this should fix the test failures https://github.com/ManageIQ/ui-components/pull/450